src/pages/Login/Login.scss

```
@@ -0,0 +1,77 @@
+@import '../../styles/variables';
+
+.login {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 100px 0px;
+
+  input::placeholder {
+    color: #dddddd;
+  }
+
+  .loginBox {
+    width: 400px;
+    height: 60px;
+    margin-bottom: 15px;
+    padding-left: 15px;
+    border: 1px solid #dddddd;
+    border-radius: 5px;
+  }
```

```suggestion
  .loginBox {
    width: 400px;
    height: 60px;
    margin-bottom: 15px;
    padding-left: 15px;
    border: 1px solid #dddddd;
    border-radius: 5px;
    
     &::placeholder {
    color: #dddddd;
    }
  }
```

태그에 직접적으로 스타일을 주는 것은 차후 스타일 충돌을 일으킬 가능성이 있고,
이미 loginBox라는 선택자는 input 태그기 때문에 이렇게 스타일을 부여할 수 있겠네요~